src/main/puzzle/piece/piece-speeds.gd

```diff
@@ -63,9 +63,9 @@ func _ready() -> void:
 	_add_speed(PieceSpeed.new("AF", 20*G, 16, 12, 7, 10, 30, 12,  6))
 	
 	# crazy; 120-250 blocks per minute
-	_add_speed(PieceSpeed.new( "F0",    4, 12, 8, 7, 10, 24, 6, 3))
-	_add_speed(PieceSpeed.new( "F1",  1*G, 12, 8, 7, 10, 24, 6, 3))
-	_add_speed(PieceSpeed.new( "FA", 20*G, 12, 8, 7, 10, 24, 6, 3))
+	_add_speed(PieceSpeed.new( "F0",    4, 12, 8, 5, 10, 24, 6, 3))
+	_add_speed(PieceSpeed.new( "F1",  1*G, 12, 8, 5, 10, 24, 6, 3))
+	_add_speed(PieceSpeed.new( "FA", 20*G, 12, 8, 5, 10, 24, 6, 3))
```

Comment:
What are these three '10' numbers near the end that didn't change? Is it related to DAS?

Reply:
Those '10 numbers were the DAS; I changed the prelock by mistake.

I've still reduced the prelock for consistency, but changed the DAS from 10/7 to an average 8.